### PR TITLE
added age calculator application using python

### LINKED ISSUE #732: Implement age calculator in python
> ### I'm submitting a
> * [ ]  bug report.
> * [x]  feature request.
> 
> ### About
> I would like to implement age calculator using different python modules with GUI python tkinter .
This application can be used anywhere for example (in forms) by taking birth date and current date as input it will give the current age of person as output.
> 
> ### Do you want to work on this issue?
> YES

Screenshot (68)

### PR DESCRIPTION
**Pull request by anandalisha**
## Explanation

  This application can be used anywhere for example (in forms) by taking birth date and current date as input it will give the current age of person as output. I have implemented this application using various python modules with GUI python tkinter .

List out the libraries imported: tkinter, datetime

Fixes #732 

## Checklist

- [x] The PR title starts with "Fix #issue_number".  
- [x] My branch is up-to-date with the main branch.
- [x] The PR does not contain any unnecessary auto-generated code.
- [x] The PR does not break any part of the existing code.
- [x] The PR creator have carefully read & adher to the guidelines of the project.
- [x] The PR is raised after the issue has been assigned to the PR creator.

## Output
Screenshot (71)

### CODE REVIEW

**Comment by anandalisha** [comment]
@GowthamGoush @RaveenaBhasin please review PR

**Review by GowthamGoush — CHANGES_REQUESTED** [review]
@anandalisha, please consider removing the changes that you've made to the files irrelevant to this script.

File: Python/Age Calculator/Age_Calculator_GUI.py [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+
+# The Driver Code
+if __name__ == '__main__':
+
+    # Creating the GUI window
+    root = Tk()
+
+    # Setting the background color
+    root.config(background='pink')
```
Comment (mehabhalodiya) [inline Python/Age Calculator/Age_Calculator_GUI.py:95]: I suggest you, not to give any `color` attribute.
It will look good the default one!

**Review by mehabhalodiya — COMMENTED** [review]
Default background is good enough, I think!

File: Python/Age Calculator/Age_Calculator_GUI.py (hunk 1 above)
Comment (GowthamGoush) [reply Python/Age Calculator/Age_Calculator_GUI.py:95]: Sorry, I did not mention anything regarding the background. In fact, you have deleted around 613 files in the repo, so I wanted you to revert that back.

Comment (mehabhalodiya) [reply Python/Age Calculator/Age_Calculator_GUI.py:95]: I have deleted anything?

Comment (anandalisha) [reply Python/Age Calculator/Age_Calculator_GUI.py:95]: @mehabhalodiya  do you want me to remove background color?

**Comment by mehabhalodiya** [comment]
@Tejas1510 Please have a look!
I think that the default background color of white will be good instead of providing in `color` attribute.

File: Python/Age Calculator/Age_Calculator_GUI.py (hunk 1 above)
Comment (GowthamGoush) [reply Python/Age Calculator/Age_Calculator_GUI.py:95]: > I have deleted anything? @GowthamGoush

Yeah, in your final commit, around 613 files were deleted, so consider resetting the head to one commit before.

**Comment by anandalisha** [comment]
@GowthamGoush  @Tejas1510  @RaveenaBhasin  Please have a review of the PR

**Review by GowthamGoush — APPROVED** [review]
Everything is fine now, and thanks for your patience. @mehabhalodiya, may I go ahead in merging this PR?

**Comment by mehabhalodiya** [comment]
> Everything is fine now, and thanks for your patience. @mehabhalodiya, may I go ahead in merging this PR?

Sure!

**Pull request merged**